jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/content/PathContentSource.java

```diff
@@ -40,20 +42,23 @@ public class PathContentSource implements Content.Source
     private final SerializedInvoker invoker = new SerializedInvoker();
     private final Path path;
     private final long length;
-    private final ByteBufferPool byteBufferPool;
-    private int bufferSize = 4096;
-    private boolean useDirectByteBuffers = true;
+    private ByteBufferPool.Sized byteBufferPool;
```

Comment:
Ditto: making this field non-final smells like a bad idea.

Reply:
Ditto - can we smell bad until we remove the setters in 12.1?

Reply:
OK.